server/models/ChartsModel/index.js

```diff
@@ -30,27 +33,30 @@ const ChartsModel = {
       ),
 }
 
-const chartsListQuery = (user) => {
+const chartsListQuery = (user, queryParams) => {
   const { uid, favoriteCharts } = user
+  const searchQuery = queryParams?.searchedCharts
```

Comment:
what do you think about using a more generic term like `search` instead of `searchedCharts`? We're already in the charts model so can we assume the charts part?

Reply:
[216b66e](https://github.com/AMP-SCZ/dpdash/pull/560/commits/216b66e967a28fae64661657855e58bd0984fef3)

updated